packages/grafana-ui/src/components/Portal/Portal.tsx

```diff
@@ -11,21 +11,21 @@ interface Props {
 export function Portal(props: PropsWithChildren<Props>) {
   const { children, className, root = document.body, forwardedRef } = props;
   const theme = useTheme2();
-  const [node] = useState(document.createElement('div'));
+  const [node] = useState(() => document.createElement('div'));
```

Comment:
I find it a bit odd to initialize `node` using useState when we don't need the setState functionality. Could we instead use `useRef`? Something like this?
```
const node = useRef(null)
if (!node.current) {
 node.current = document.createElement('div')
}
```